media/js/firefox/new/scene2-email.js

```diff
@@ -0,0 +1,52 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+(function($) {
+    'use strict';
+
+    var isIELT9 = window.Mozilla.Client.platform === 'windows' && /MSIE\s[1-8]\./.test(navigator.userAgent);
```

Comment:
All this JS looks the same as the regular `scene2.js` code. Rather than have to maintain all this JS twice, I'd suggest putting the new modal-related JS code into a separate file that we can include in the template, in addition to the regular `scene2.js` file?

Reply:
Good call. It is indeed just a copy of scene2.js with the modal added, no reason for the repetition. I'll update.